cabal-install/changelog

```diff
@@ -16,6 +16,7 @@
 	  --package-env` (#6298)
 	* `cabal v2-run` works with `.lhs` files (#6134)
 	* `subdir` in source-repository-package accepts multiple entries (#5472)
+	* 'cabal install' shows the resulting executable full path (symlinking and copying) (#6575)
```

Comment:
It won't be in `3.2.0.0`. Omit changelog entry, I'll add it afterwards myself.

Reply:
Removed as requested